app/screens/server/header.tsx

```diff
@@ -27,7 +27,7 @@ const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => ({
         ...typography('Heading', 400, 'SemiBold'),
     },
     connect: {
-        width: 270,
+        width: 320,
```

Comment:
I'm also curious about this

Reply:
this was because in the server screen, the title was not getting displayed correctly, by adding more space we fix it without affecting how it looks in the English original design.

Reply:
ohhh yes I remember this, we made this change together cause of the way it was shown in languages other than english